grakn-engine/src/main/java/ai/grakn/engine/GraknEngineServer.java

```diff
@@ -192,245 +117,42 @@ private void lockAndInitializeSystemSchema() {
             if (lock.tryLock(60, TimeUnit.SECONDS)) {
                 loadAndUnlock(lock);
             } else {
-                LOG.info("{} found system schema lock already acquired by other engine", this.engineId);
+                LOG.info("{} found system schema lock already acquired by other engine");
```

Comment:
Maintaining the engine id here and elsewhere might be useful for debugging on the cluster.

Reply:
you're right! I thought it was worth to remove, because basically this class depends already on so many other classes ..

```java
GraknEngineServer(GraknEngineConfig prop, TaskManager taskManager, EngineGraknTxFactory factory, LockProvider lockProvider, GraknEngineStatus graknEngineStatus, RedisWrapper redisWrapper, ExecutorService taskExecutor, HttpHandler httpHandler)
```

Just let me know if you think it's worth, otherwise I will revert back!

Reply:
Hmmm not sure. @pluraliseseverythings mostly debugs engine. How helpful is knowing the engine ID?

Reply:
I imagine when we're working with many nodes in a cluster it will useful, especially when logs also displays engine ID.